.github/workflows/run-tests.yml

```diff
@@ -62,6 +62,7 @@ env:
   PR_BASE_COMMIT: ${{ github.event.pull_request.base.sha }}
   DOCKER_COMPOSE_FILE: ./develop/github/docker-compose.yml
   TEMPORAL_VERSION_CHECK_DISABLED: 1
+  MAX_TEST_ATTEMPTS: 3
```

Comment:
Used to verify if the job crashed or not. Also, using "attempt" instead of "retries" now consistently (makes math easier).